taxi_service/settings.py

```diff
@@ -132,4 +132,8 @@
 # Default primary key field type
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
+LOGIN_REDIRECT_URL = "/"
```

Comment:
Make sure to add a blank line at the end of the file as per the checklist requirement.